src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/ClusterResourcesConfig.scala

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.dsde.workbench.leonardo.config
+
+case class ClusterResourcesConfig(
+  configFolderPath: String,
```

Comment:
I mentioned this in a different PR, but what do people think about having better-typed config classes? For instance instead of a `configFolderPath: String` and `initActionsFileName: String` we could have a `initActionsFile: File`.

Reply:
I can add these